### PR TITLE
Annual report 2019-2020

### LINKED ISSUE #908: Publish first annual report
Our first financial year ended in Dec 2020. We now need to publish our first annual report.

We currently plan to publish:
- a narrative annual report aimed at a broader audience
- a financial report as required by Dutch non-profit (ANBI) law

We'll publish this in 2 formats:
- Markdown, for easy reading and access
- PDF, for higher impact design and more engaging presentation

We aim to publish this in mid-April, well before the Dutch gov't deadline of July.

### PR DESCRIPTION
**Pull request by ElenaFdR**
Still to do:

- [x] create index page
- [x] link finance PDF
- [x] add finance markdown
- [x] add final date for signatures in financial MD
- [x] link narrative PDF
- [x] add narrative markdown

This closes #908 and partially fulfils #311.

### CODE REVIEW

File: organization/annual-reports/financial-report-2019-2020.md [hunk 1]
```diff
@@ -187,4 +187,200 @@ Gifts may only be accepted without prior approval when:
...
+
+* Dia Holding B.V. (Arnout Schuijff)
+* DC Butterfield and AD Rubio Memorial Foundation (Stewart Butterfield)
+* Elkes Foundation (Josh Elkes)
+
+These philanthropists share our commitment to our mission. To prevent undue influence, all donations are un-earmarked and go into our general operating budget, in compliance with our status as a Dutch public benefit organization (ANBI).
+
+During this start-up phase, we try to look two years ahead with our financial planning.  The organization  has low financial risk: there are no debts and no assets. There is a loan agreement (anti-kraak) for the office, which incurs no rent. In the second quarter of 2020, we transitioned to remote-first working in response to government recommendations during the Covid-19 crisis. When the loan agreement expires at the end of June 2021, the contract can be extended on a monthly basis until a tenant is found. All our activities are location independent, meaning the organization can continue to function in a remote-first context. Nevertheless, some budget has been set aside starting from the third quarter of 2021 in case a new office space is needed.
```
Comment (ericherman) [inline organization/annual-reports/financial-report-2019-2020.md:206]: Nit-pick: extra space in "The organization  has low financial risk" .... not a blocker at all, as it won't render any differently in a browser.

Comment (ericherman) [reply organization/annual-reports/financial-report-2019-2020.md:206]: (there are a few other double-spaces in the middle of sentences on the page as well ... again, not a big deal)

Comment (ElenaFdR) [reply organization/annual-reports/financial-report-2019-2020.md:206]: Done, thanks!

File: organization/annual-reports/annual-report-2019-2020.md [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+
+### Prove our impact
+
+Infrastructure is the long-term foundation for more immediate work, and as such tends to be taken for granted. Similarly, the infrastructural value we provide through codebase stewardship or framework definition (like the Standard for Public Code) and analysis can go unnoticed even by those who could not imagine their work without it.
+
+High-profile contributions of work, resources or funding help signal our value. To attract and retain members, additional maintainers, and philanthropic donors, we need to better articulate the value we add to public sector digital transformation. In the coming year, we’ll produce more case studies, business cases and testimonials.
+
+### Professionalize an exemplary open public organization
+
+Our vision of a flourishing ecosystem of public code contains inherent assumptions about how a responsible and professional public organization behaves. This includes learning in the open and having strong, accountable processes—not just at the codebase level, but across the organization.
```
Comment (Ainali) [inline organization/annual-reports/annual-report-2019-2020.md:469]:
```suggestion
Our vision of a flourishing ecosystem of public code contains inherent assumptions about how a responsible and professional public organization behaves. This includes learning in the open and having strong, accountable processes — not just at the codebase level, but across the organization.
```

File: organization/annual-reports/annual-report-2019-2020.md [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+After Italy’s Team Digitale successfully based their national public code repository on our jointly developed publiccode.yml standard, the German and Dutch public code communities are looking to do the same.
+
+#### Adoption as a UN Digital Public Good?
+
+Building on our [public commitment](https://blog.publiccode.net/policy/2020/10/02/we-support-the-sustainable-development-goals.html) to the Sustainable Development Goals in September 2020, we proposed the Standard for Public Code for adoption as a Digital Public Good. While our application approval is pending, we’re proud that the Standard is [recommended to other aspiring Digital Public Good projects](https://github.com/DPGAlliance/DPG-Resources).
+
+#### More involvement
+
+* Gave feedback on a draft of Open Source Observatory and Repository’s [Guidelines for Sustainable Open Source Communities in the Public Sector](https://joinup.ec.europa.eu/collection/open-source-observatory-osor/news/supporting-oss-communities-public-sector)
+* Contributed to the State of California’s ‘[Vision 2023’ Technology Strategic Plan](https://vision2023.cdt.ca.gov/)
```
Comment (Ainali) [inline organization/annual-reports/annual-report-2019-2020.md:253]:
```suggestion
* Contributed to the State of California’s [‘Vision 2023’ Technology Strategic Plan](https://vision2023.cdt.ca.gov/)
```

**Comment by Ainali** [comment]
In five places we write "open-source" and in twenty "open source". I guess we should only use the latter?

**Review by ericherman — DISMISSED** [review]
I think the content is fine, and I didn't see anything that looked like an error nor could easily be misunderstood.

Even the tiny nit-picks I think are not worth fussing over, e.g.: I'm not used to seeing paragraphs start with a space in Markdown, because I don't think it will matter when displayed.

File: organization/annual-reports/financial-report-2019-2020.md [hunk 4]
```diff
@@ -0,0 +1,645 @@
...
+
+The current financial situation gives the organization financial security until April 2022. We expect to receive both, more member contributions, as well as more philanthropic gifts. However, it is possible that the income will be insufficient to cover our operational expenses. Costs will then have to be drastically reduced. Since personnel costs comprise the majority of our costs, reductions will need to be made in this area.
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply organization/annual-reports/financial-report-2019-2020.md]: Good spot!

**Comment by ElenaFdR** [comment]
Request for specific attention: the link to /annual-reports on organization/index doesn't work, and I don't understand why not.

**Comment by bvhme** [comment]
@ElenaFdR, because the path is not relative but root based with the `/` in front of it.

Further reading: https://learn-the-web.algonquindesign.ca/topics/paths-folders/#root-relative

**Review by demkodo — APPROVED** [review]

File: organization/annual-reports/annual-report-2019-2020.md [hunk 5]
```diff
@@ -0,0 +1,486 @@
...
+For now, we seem to be on the front line of a big change: the government of the future will be open.
+
+Thanks to our community, expertise, momentum and team, we’re uniquely positioned to go at these challenges head on. And we’re excited to do so.
+
+I hope you’ll be joining us!
+
+**Boris van Hoytema**
+Chief executive, Foundation for Public Code
+4 May, 2021
+Amsterdam, Netherlands
```
Comment (bvhme) [inline organization/annual-reports/annual-report-2019-2020.md:486]: This now is on one rule, I propose:

```suggestion
**Boris van Hoytema**,
Chief executive, Foundation for Public Code

4 May 2021 — Amsterdam, Netherlands
```

File: organization/annual-reports/annual-report-2019-2020.md [hunk 6]
```diff
@@ -0,0 +1,486 @@
...
+
+In the coming years, the Foundation will provide awareness, education, and share our proven stewardship approach through media, symposia, and press activities. We believe a mature public code ecosystem, in which communities of public organizations around the globe collaborate on codebases that provide all variety of infrastructural services, will fundamentally transform the relationship between citizens and their governments. Dynamically adaptive participatory frameworks for urban planning, spatial computing, social identity and communication, and public media consumption and production can all be made more inclusively available and accountably open. Traditional urban institutions like the transit network, the library, and the community center can become new vitally network-enabled public platforms.
+
+In a world where the mission of the Foundation for Public Code is fully realized, the process of governance itself becomes more responsive, resilient, and better prepared for the challenges we all face in the 21st century.
+
+**Ben Cerveny**
+President, Foundation for Public Code
+4 May, 2021
+Amsterdam, Netherlands
```
Comment (bvhme) [inline organization/annual-reports/annual-report-2019-2020.md:42]: This is now on 1 rule, I propose

```suggestion
**Ben Cerveny**,
President, Foundation for Public Code

4 May, 2021 — Amsterdam, Netherlands
```

**Review by demkodo — DISMISSED** [review]

File: organization/annual-reports/annual-report-2019-2020.md [hunk 7]
```diff
@@ -0,0 +1,486 @@
...
+
+#### Grew and recruited from a deep talent pool
+
+We doubled our initial team to ten brilliant colleagues, including experts in municipal collaboration, community building, open source development and digital transformation in government, and are supported by three consultants boosting our capabilities in design, illustration and finance.
+
+#### Secured funding (with more lined up for the future)
+
+We wouldn’t be where we are today without our donors, who believe we are integral to the structural solution to scale open source software in public organizations. Generous donations from **Arnout Schuijff** (co-founder, Adyen), **Stewart Butterfield** (CEO, Slack Technologies) and **Josh Elkes** (CEO, Harbour) enabled us to develop our stewardship operations, conduct research and development, and raise awareness around public code. As a public benefit organization and with the KBFUS Public Code US fund, gifts in both the Netherlands and the United States are tax deductible.
+
+> Our donors pledged nearly €3 million in 2019 and 2020.
```
Comment (bvhme) [inline organization/annual-reports/annual-report-2019-2020.md:107]: This is not a quote

```suggestion
Our donors pledged nearly €3 million in 2019 and 2020.
```

If it requires emphasis, consider using

```suggestion
**Our donors pledged nearly €3 million in 2019 and 2020.**
```

File: organization/annual-reports/annual-report-2019-2020.md [hunk 8]
```diff
@@ -0,0 +1,486 @@
...
+
+##### Results for OpenZaak
+
+* OpenZaak maintainers consider the Standard for Public Code integral to achieving their vision
+* Joint codebase governance by vendors and municipalities through established technical and product steering groups
+* Community independently organizes and conducts technical and product steering meetings
+* Established and documented process for security incidents
+* A new product website in both English and Dutch
+
+>[See the current status for OpenZaak](https://publiccode.net/codebases/openzaak.html)
```
Comment (bvhme) [inline organization/annual-reports/annual-report-2019-2020.md:216]: This is not a quote and inconsistent with similar content

```suggestion
[See the current status for OpenZaak](https://publiccode.net/codebases/openzaak.html)
```

File: organization/annual-reports/annual-report-2019-2020.md [hunk 9]
```diff
@@ -0,0 +1,486 @@
...
+
+Generous philanthropic donations have allowed us to expand our stewardship operations, conduct research and development, and raise awareness around public code. Our donors pledged a total of €2,983,857 in 2019 and 2020. To prevent undue influence, all donations are un-earmarked and go into our general operating budget, in compliance with our status as a Dutch public benefit organization (ANBI).
+
+During this startup phase, we try to look two years ahead with our financial planning. The organization has low financial risk: it is not capital intensive (no large capital expenditures) and has no loans with regard to the organization’s assets.
+
+The current financial situation gives the Foundation for Public Code financial security until April 2022.
+
+**Our largest cost by far (really, our biggest investment) is the amazing people who make our work possible. This includes their salaries, wage taxes, insurances and expenses.**
+
+![80% people](https://raw.githubusercontent.com/publiccodenet/about/annual-report-2019-2020/organization/annual-reports/images/80per-people.png)
```
Comment (bvhme) [inline organization/annual-reports/annual-report-2019-2020.md:333]: I noticed this graphic only works on white, a potential risk in future reuse, but not an issue for now.

File: organization/annual-reports/annual-report-2019-2020.md [hunk 10]
```diff
@@ -0,0 +1,486 @@
...
+
+In June 2020, the Dutch chapter of the Internet Society (ISOC) recognized our work with the 2020 Aanmoedigingsprijs (Encouragement Award). We were proud to be nominated for the annual Innovation Award, and delighted by Internet Society Netherlands’ recognition.
+
+> "The Foundation for Public Code is helping our collaboration of Dutch cities to increase our code quality, and making codebases more accessible for others to use" — **Jacco Brouwer**, the association of Dutch municipalities (VNG-Realisatie)
+
+>“OS2’s board and secretariat have a continued focus on international cooperation, and the exporting of OS2 open source products for use outside Denmark. In that matter, we wish to work with the Foundation for Public Code, as we consider it a very strong bid for the best cross-governmental collaboration partner at the European level.
+>We are well aware that for the OS2 portfolio to perform internationally, demands will be higher—and this is not a task we can fulfil ourselves. Cooperation with the Foundation for Public Code can increase the international viability of our open source products.” — **Rasmus Frey**, Chief Executive, OS2 (Danish Public Digitalization Network)
```
Comment (bvhme) [inline organization/annual-reports/annual-report-2019-2020.md:274]:
```suggestion
> “OS2’s board and secretariat have a continued focus on international cooperation, and the exporting of OS2 open source products for use outside Denmark. In that matter, we wish to work with the Foundation for Public Code, as we consider it a very strong bid for the best cross-governmental collaboration partner at the European level.
> We are well aware that for the OS2 portfolio to perform internationally, demands will be higher—and this is not a task we can fulfil ourselves. Cooperation with the Foundation for Public Code can increase the international viability of our open source products.” — **Rasmus Frey**, Chief Executive, OS2 (Danish Public Digitalization Network)
```

**Review by bvhme — CHANGES_REQUESTED** [review]
General:

- Do not use the Github URLs for the images, those URLs are not meant for use on other sites and will throttle. Use the relative links instead.

File: organization/annual-reports/annual-report-2019-2020.md (hunk 9 above)
Comment (demkodo) [reply organization/annual-reports/annual-report-2019-2020.md:333]: should we create an issue to ensure it's tackled in the future?

Comment (ElenaFdR) [reply organization/annual-reports/annual-report-2019-2020.md:333]: Thanks, good spot. @demkodo, please tag @angelplasma in the issue?

**Review by bvhme — APPROVED** [review]

**Pull request merged**